src/js/edu-benefits/1995/config/form.js

```diff
@@ -34,7 +32,6 @@ const {
   educationObjective,
   nonVaAssistance,
   reasonForChange,
-  email,
```

Comment:
It's still being used in the personal info definition. The schema definition for it is just a string with a format of `email`, so it's not a shared definition in vets-json-schema.